crates/nu-command/tests/commands/math/mod.rs

```diff
@@ -282,6 +282,66 @@ fn modulo() {
     assert_eq!(actual.out, "1");
 }
 
+#[test]
+fn unit_multiplication_math() {
+    let actual = nu!(
+        cwd: "tests/fixtures/formats", pipeline(
+        r#"
+            1mb * 2
+        "#
+    ));
+
+    assert_eq!(actual.out, "1.9 MiB");
```

Comment:
Unrelated to the PR but that seems like an annoying rounding behavior (and with 2 that is definitely not a float problem). Also mentioned in https://github.com/nushell/nushell/issues/6533

Reply:
What's going on here is when `1mb * 2` is parsed, it finds that it matches this:
```
        (Unit::Megabyte, "MB", Some((Unit::Kilobyte, 1000))),
```
falls through some code and then eventually does `1 * 1000` and then adds the `Unit::Kilobyte` from above (in nu-parser/src/parser.rs). Then `1000KB` gets converted to the representation of bytes on your system via `filesize_metric` and `filesize_format` in the `format_filesize()` function (in nu-protocol/src/value/mod.rs.

Mostly logging all this information here for future changes. I'm not sure how to fix the 1.9 MiB because of filesize_metric and filesize_format, but we can fix the 3.14sec reporting as 3sec 142ms, which i'm working on now.